src/java/ImageUpload/UserImageCollection.java

```diff
@@ -109,13 +111,13 @@ public static void create(Connection conn, File zippedFile, User uploader, Manus
       File[] images = getAllJPGsRecursive(dir);
       for (int i = 0; i < images.length; i++) {
          if (!validateImage(images[i])) {
-            System.out.print("bad image "+images[i].getName()+", would do something\n");
+                out.print("bad image "+images[i].getName()+", would do something\n");
```

Comment:
I like leaving these like `System.out` because `out` is often used as a variable to handle things like `Buffers`, `Writers`, `PrintWriters`, `OutputStreams` and various `Response`s.  In general, System is the one you don't import so that `System.out.println`s and `out.write()` style things don't become jumbled in the code.  Too often we absorb Libraries or use third part code snippets, so be careful making this a habit.

Reply:
That makes a lot of sense to me. I removed a lot of `System.out`s that were commented or obvious from bug-chasing and not useful flags. I'm sure the Netbeans refactor swept up the `System.`s without my noticing, but I will watch it.

My preference would actually be towards using `Logger` better so that instead of commenting out all these lines, we can just run with a different level of verbosity.

Reply:
That makes a lot of sense to me. I removed a lot of `System.out`s that were commented or obvious from bug-chasing and not useful flags. I'm sure the Netbeans refactor swept up the `System.`s without my noticing, but I will watch it.

My preference would actually be towards using `Logger` better so that instead of commenting out all these lines, we can just run with a different level of verbosity.